packages/build/src/services/bundler/vendor.ts

```diff
@@ -11,13 +11,41 @@ export interface VendorBundleServiceEvents extends OutputServiceEvents {
   [BundleVendorErrorEvent]: () => void
 }
 
+const includesDefault = [
+  'react',
+  'react-dom',
+  'react-helmet',
+  'lodash',
+  'typestyle',
+]
+
 export class VendorBundleService {
+  static get includes() {
+    try {
+      const json = require(`${process.cwd()}/package.json`)
+      if (Array.isArray(json.vendorBundleIncludes)) {
+        return json.vendorBundleIncludes
+      }
+      const add = json.vendorBundleIncludes.add as string[] | undefined
+      const remove = json.vendorBundleIncludes.remove as string[] | undefined
+      const includesSet = new Set<string>(includesDefault)
+      if (Array.isArray(add)) {
+        add.forEach(a => includesSet.add(a))
+      }
+      if (Array.isArray(remove)) {
+        remove.forEach(r => includesSet.delete(r))
+      }
+      return Array.from(includesSet)
+    } catch (e) {
+      return includesDefault
```

Comment:
what about a printed warning?
I personally would let it crash

Reply:
as there is a default, it's not necessary to specify this in every project. we could include a warning in case the entry is faulty though ...